yaroslavdu/meshbit_fw
# Button to GPIO35; Triple click --> SOS to channel 2

This change moves the button from GPIO38 (the central button on TBeam board) to GPIO35, allowing connection of an external button.

Also, the change disables GPS power management via a triple click, instead implementing posting of the "SOS" text message to meshtastic channel 2 (defined by `SOS_SIGNAL_MESHTASTIC_CHANNEL_NUM` in the `variant.h`).

## Testing

* add an external button between GPIO35 and GND
* build & flash
* configure at least 2 meshtastic devices with channel 2 enabled and having the same channel name and PSK
* after devices are boot up and functional, do a triple press on the external button
* observe "SOS" text message on other devices
